pkg/k8s/labels/labels.go

```diff
@@ -32,6 +35,9 @@ const (
 	// DeploymentAnnotation indicates the original deployment manifest  when the development container was activated
 	DeploymentAnnotation = "dev.okteto.com/deployment"
 
+	// TimestampAnnotation indicates the timestamp of an operation
+	TimestampAnnotation = "dev.okteto.com/timestamp"
```

Comment:
can we rename this to something more descriptive dev.okteto.com/last-updated or something like that?